spec/requests/pages_controller_spec.rb

```diff
@@ -0,0 +1,99 @@
+require "rails_helper"
+
+RSpec.describe PagesController, clamav: true do
```

Comment:
As you've added the tests, can you remove the simplecov exclusion? https://github.com/ministryofjustice/laa-apply-for-legal-aid/blame/main/.simplecov#L10

Reply:
oh, i thought i had. will do.

Reply:
yeah, i had. https://github.com/ministryofjustice/laa-apply-for-legal-aid/pull/6100/files#diff-24002d905f0809599eae02b5cd7b58f8821aa89272b2b207c3e589b7a7a4f3a6L10


will remove first commit now though

Reply:
Huh! How did I miss that! 🤦‍♂️